src/client.ts

```diff
@@ -72,24 +94,31 @@ export interface IMetadata {
      */
     title?: string;
 
+    // Supports arbitrary string keys.
     [key: string]: any;
 }
 
 /**
- * A single Documentalist page, parsed from a single source file.
+ * The output of `renderBlock` which parses a long form documentation block into
+ * metadata, rendered markdown, and tags.
  */
-export interface IPageData {
-    /** Absolute path of source file. */
-    absolutePath: string;
+export interface IBlock {
+    /** Parsed nodes of source file. An array of markdown-rendered HTML strings or `@tag` objects. */
+    contents: StringOrTag[];
 
     /** Raw unmodified contents of source file (excluding the metadata). */
-    contentRaw: string;
-
-    /** Parsed nodes of source file. An array of rendered HTML strings or `@tag` objects. */
-    contents: StringOrTag[];
+    contentsRaw: string;
```

Comment:
i definitely prefer consistency. either `content` or `contents`. i see no need to use them both in the same interface.